ONSdigital/dp-frontend-renderer
# Add logic whether to render edition name or not

### What

Update template to use ShowEditionName boolean to determine whether to show edition title or not

### How to review

To test you'll need:
https://github.com/ONSdigital/dp-frontend-models/pull/36
https://github.com/ONSdigital/dp-frontend-dataset-controller/pull/111

Go to version page for an edition with title "time-series" (e.g. `datasets/<<id>>/edition/time-series/versions/<<id>>`) and the edition title shouldn't show. Check with an edition not called "time-series" and it should appear. 

### Who can review

Anyone
